siliconcompiler/tools/openroad/sc_floorplan.tcl

```diff
@@ -56,49 +56,155 @@ if {[dict exists $sc_cfg pdk $sc_pdk aprtech openroad $sc_stackup $sc_libtype tr
   make_tracks
 }
 
+set do_automatic_pins 1
 if { 0 } {
```

Comment:
Is `if { 0 }` different from `if False:`?

Reply:
It's the same in tcl. The `if {0}` is a placeholder for the padring code to make sure it doens't get included for now